internal/pkg/service/stream/sink/tablesink/storage/level/local/reader/volume/reader.go

```diff
@@ -26,7 +28,14 @@ func (v *Volume) NewReaderFor(slice *storage.Slice) (r reader.Reader, err error)
 	}
 
 	// Setup logger
-	logger := v.logger
+	logger := v.logger.With(
+		attribute.String("projectId", slice.ProjectID.String()),
```

Comment:
I update the code in the storage layer, log messages now contain attributes.

See the next commit with tests.

There is a long-lived logger, and I want to log the attributes with all operations, regardless of context, therefore I am not using `ctxattrs` (they are better for `userID` and similar - request based - attributes).